models/comments-model.js

```diff
@@ -45,3 +45,19 @@ exports.createComments = (articleId, { username, body }) => {
       return comment;
     });
 };
+
+exports.removeComment = (commentId) => {
+  if (!Number.isInteger(Number(commentId))) {
```

Comment:
You have error handlers for this. No need to custom reject

Reply:
<img width="718" alt="Screenshot 2024-08-27 at 14 19 44" src="https://github.com/user-attachments/assets/fcdf08e2-81d9-4373-805f-d74fbb7f3a03">